src/r2mm/mods/ProfileImportExport.ts

```diff
@@ -25,17 +25,26 @@ const getDownloadDestination = async (): Promise<string> => {
     return path.join(cacheDir, "import.r2z");
 }
 
-const isProfileDataValid = (profileData: string): boolean => {
-    return profileData.startsWith(PROFILE_DATA_PREFIX)
+
+const B64_REGEX = /[A-Za-z0-9+/=]/;
+
+const isValidBase64 = (profileData: string): boolean => {
```

Comment:
Nit but this would be more consistent were it defined as a function instead. I don't think there's any real difference between the two other than hoisting but it at least makes it clear that it's a function rather than a variable